ast/ast.go

```diff
@@ -1662,6 +1701,49 @@ type CreateRole struct {
 	Name   *Ident
 }
 
+// CreateChangeStream is CREATE CHANGE STREAM statement node.
+//
+// CREATE CHANGE STREAM {{.Name | sql}}
+// {{ if or len(.Watch) > 0 .WatchAll }}
+// FOR {{ .WatchAll }}ALL{{ else }}{{ .Watch | sqlJoin ","}}{{ end }}
+// {{ end }}
+// {{ if .Options }}
+// OPTIONS {{ .Options | sqlJoin ","}}
+// {{ end }}
+type CreateChangeStream struct {
+	// pos = Create
+	// end = {{ if .Options }}
+	//          .Options.end
+	//       {{ else }}
+	//         {{ if len(.Watch) > 0 }}
+	//           .Watch[len(.Watch)-1].end
+	//         {{ else if .WatchAll }}
+	//           "ALL".end
+	//         {{ else }}
+	//           .Name.end
+	//         {{ end }}
+	//       {{ end }}
+	Create   token.Pos // position of "CREATE" keyword
+	Name     *Ident
+	Watch    []*ChangeStreamWatch
+	WatchAll bool
+	Options  []Expr
+}
+
+// Watch is tables and columns that are watched by change streams.
+//
+//	{{.TableName}} {{if .Columns}}({{ .Columns | sqlJoin ","}}){{end}}
+type ChangeStreamWatch struct {
+	// pos = TableNamePos
+	// end = {{if .Columns }}{{ Rparen + 1 }}{{else}}.TableName.end{{end}}
+
+	TableNamePos   token.Pos // position of this watch definition
+	TableName      *Ident
+	Columns        []*Ident
+	Lparen, Rparen token.Pos // position of "(" and ")"
```

Comment:
`Lparen` is unnecessary because it is used for neither `pos` nor `end`.

Reply:
Also, `token.Pos` fields are put into the top of fields and separated by a newline to other fields. See other definitions.